ui-components/src/helpers/tableSummaries.tsx

```diff
@@ -1,43 +1,34 @@
 import * as React from "react"
 import { t } from "./translator"
-import { UnitSummary } from "@bloom-housing/backend-core/types"
+import { UnitsSummary } from "@bloom-housing/backend-core/types"
 import { GroupedTableGroup } from "../tables/GroupedTable"
 
-export const unitSummariesTable = (summaries: UnitSummary[]) => {
-  const unitSummaries = summaries.map((unitSummary) => {
+export const unitSummariesTable = (summaries: UnitsSummary[]) => {
+  const unitsSummaries = summaries.map((unitSummary) => {
     const minIncome =
-      unitSummary.minIncomeRange.min == unitSummary.minIncomeRange.max ? (
-        <strong>{unitSummary.minIncomeRange.min}</strong>
+      unitSummary.minimumIncomeMin == unitSummary.minimumIncomeMax ? (
+        <strong>{unitSummary.minimumIncomeMin}</strong>
       ) : (
         <>
-          <strong>{unitSummary.minIncomeRange.min}</strong> {t("t.to")}{" "}
-          <strong>{unitSummary.minIncomeRange.max}</strong>
+          <strong>{unitSummary.minimumIncomeMin}</strong> {t("t.to")}{" "}
+          <strong>{unitSummary.minimumIncomeMax}</strong>
         </>
       )
 
-    const getRent = (rentMin: string, rentMax: string, percent = false) => {
+    const getRent = (rent: string, percent = false) => {
       const unit = percent ? `% ${t("t.income")}` : ` ${t("t.perMonth")}`
-      return rentMin == rentMax ? (
+      return (
         <>
-          <strong>{rentMin}</strong>
-          {unit}
-        </>
-      ) : (
-        <>
-          <strong>{rentMin}</strong> {t("t.to")} <strong>{rentMax}</strong>
+          <strong>{rent}</strong>
           {unit}
         </>
       )
     }
 
     // Use rent as percent income if available, otherwise use exact rent
-    const rent = unitSummary.rentAsPercentIncomeRange.min
-      ? getRent(
-          unitSummary.rentAsPercentIncomeRange.min.toString(),
-          unitSummary.rentAsPercentIncomeRange.max.toString(),
-          true
-        )
-      : getRent(unitSummary.rentRange.min, unitSummary.rentRange.max)
+    const rent = unitSummary.monthlyRentAsPercentOfIncome
+      ? getRent(unitSummary.monthlyRentAsPercentOfIncome.toString(), true)
+      : getRent(unitSummary.monthlyRent)
```

Comment:
@anders-schneider this is where I'm getting a data formating error, that I haven't totally resolved.

Reply:
It looks like `unitSummary.monthlyRent` is a `number` (https://github.com/CityOfDetroit/bloom/blob/ddf53dd92a37e5f3b3f2c96d3fe87ae009bc4efe/backend/core/src/units-summary/entities/units-summary.entity.ts#L38). Would this work:

`getRent(unitSummary.monthlyRent.toString())`